shared/src/main/scala/squants/market/Money.scala

```diff
@@ -424,30 +424,30 @@ abstract class Currency(val code: String, val name: String, val symbol: String,
 }
 
 object USD extends Currency("USD", "US Dollar", "$", 2)
-object ARS extends Currency("ARS", "Argentinean Peso", "$", 2)
-object AUD extends Currency("AUD", "Australian Dollar", "$", 2)
+object ARS extends Currency("ARS", "Argentinean Peso", "CLP$", 2)
+object AUD extends Currency("AUD", "Australian Dollar", "A$", 2)
 object BRL extends Currency("BRL", "Brazilian Real", "R$", 2)
-object CAD extends Currency("CAD", "Canadian Dollar", "$", 2)
+object CAD extends Currency("CAD", "Canadian Dollar", "Can$", 2)
 object CHF extends Currency("CHF", "Swiss Franc", "CHF", 2)
-object CLP extends Currency("CLP", "Chilean Peso", "¥", 2)
-object CNY extends Currency("CNY", "Chinese Yuan Renmimbi", "¥", 2)
+object CLP extends Currency("CLP", "Chilean Peso", "CLP$", 2)
+object CNY extends Currency("CNY", "Chinese Yuan Renminbi", "¥", 2)
 object CZK extends Currency("CZK", "Czech Republic Koruny", "Kč", 2)
 object DKK extends Currency("DKK", "Danish Kroner", "kr", 2)
 object EUR extends Currency("EUR", "Euro", "€", 2)
 object GBP extends Currency("GBP", "British Pound", "£", 2)
-object HKD extends Currency("HKD", "Hong Kong Dollar", "$", 2)
+object HKD extends Currency("HKD", "Hong Kong Dollar", "HK$", 2)
 object INR extends Currency("INR", "Indian Rupee", "₹", 2)
 object JPY extends Currency("JPY", "Japanese Yen", "¥", 0)
-object KRW extends Currency("KRW", "South Korean Won", "kr", 0)
-object MXN extends Currency("MXN", "Mexican Peso", "$", 2)
+object KRW extends Currency("KRW", "South Korean Won", "₩", 0)
+object MXN extends Currency("MXN", "Mexican Peso", "Mex$", 2)
 object MYR extends Currency("MYR", "Malaysian Ringgit", "RM", 2)
 object NOK extends Currency("NOK", "Norwegian Krone", "kr", 2)
-object NZD extends Currency("NZD", "New Zealand Dollar", "$", 2)
-object RUB extends Currency("RUB", "Russian Ruble", "руб", 2)
+object NZD extends Currency("NZD", "New Zealand Dollar", "NZ$", 2)
+object RUB extends Currency("RUB", "Russian Ruble", "\u20BD", 2)
 object SEK extends Currency("SEK", "Swedish Kroner", "kr", 2)
-object XAG extends Currency("XAG", "Silver", "oz", 4)
-object XAU extends Currency("XAU", "Gold", "oz", 4)
-object BTC extends Currency("BTC", "BitCoin", "B", 15)
+object XAG extends Currency("XAG", "Silver", "Ag", 4)
+object XAU extends Currency("XAU", "Gold", "Au", 4)
+object BTC extends Currency("BTC", "Bitcoin", "B", 15)
```

Comment:
Is there some unicode for the symbol?

Reply:
https://en.bitcoin.it/wiki/Bitcoin_symbol